examples/example_gmf_titanic.json

```diff
@@ -4,9 +4,9 @@
     "provenance": {
         "created by": {
             "name": "metasyn",
-            "version": "0.6.1.dev35+gf3115b8.d20231130"
+            "version": "0.6.1.dev45+gd3708ea.d20231212"
```

Comment:
When we make a new release, it might be nice to regenerate this file to get a clean version number here. Just a small nitpick though.

Reply:
Sure. But it's going to be changed after the new release anyway..